src/components/Layout.js

```diff
@@ -1,17 +1,24 @@
 import React from "react";
+import { BrowserRouter as Router } from "react-router-dom";
 import styles from "./Layout.scss";
 import Header from "./Header/Header";
 import Dashboard from "./Dashboard/Dashboard";
 import Sidebar from "./Sidebar/Sidebar";
+import Navigation from "./Navigation/Navigation";
 
 const Layout = () => {
   return (
-    <div className={styles.Layout}>
-      <Sidebar>
-        <Header />
-      </Sidebar>
-      <Dashboard />
-    </div>
+    <Router>
+      <div className={styles.Layout}>
+        <Sidebar>
+          <React.Fragment>
```

Comment:
To discuss: should we use `<React.Fragment>` or `<>`?

Reply:
As we've talked `<React.Fragment>` was used before - from what I see in 3 different files - so we stay with it. However, we can change it later if everybody wants to use a shorter syntax